newsfragments/3223.feature.rst

```diff
@@ -0,0 +1 @@
+Make Prometheus exporter always run for Ursula
```

Comment:
This is no longer applicable; prometheus is not mandatory anymore. Should we remove this newsfragment?